perun-base/src/main/resources/perun-base.xml

```diff
@@ -82,6 +82,7 @@
 		<property name="mfaAuthTimeout" value="${perun.introspectionEndpoint.mfaAuthTimeout}"/>
 		<property name="mfaAuthTimeoutPercentageForceLogIn" value="${perun.introspectionEndpoint.mfaAuthTimeoutPercentageForceLogIn}"/>
 		<property name="enforceMfa" value="${perun.enforceMfa}"/>
+		<property name="appAllowedRoles" value="#{'${perun.appAllowedRoles.apps}'.split('\s*,\s*')}"/>
```

Comment:
Looks like a good solution, just please include deployment notes related to the new header and new config property (an explanation of its structure would be nice)